asterisk/agi/templates/faxmail/default/en/body

```diff
@@ -0,0 +1,11 @@
+Greetings!
```

Comment:
You can use Zend_View for this if you prefer. 
http://stackoverflow.com/questions/1218191/how-can-i-make-email-template-in-zend-framework